#! /bin/sh /usr/share/dpatch/dpatch-run
## wvware-no-placeholder.dpatch by Matej Vela <vela@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: wvWare.c (wvStrangeNoGraphicData): Remove dangling reference to
## DP: placeholder.eps (#197979).

@DPATCH@
diff -urNad wv-1.2.1~/wvWare.c wv-1.2.1/wvWare.c
--- wv-1.2.1~/wvWare.c	2006-03-17 22:33:32.000000000 +0100
+++ wv-1.2.1/wvWare.c	2006-03-18 12:11:04.000000000 +0100
@@ -666,8 +666,7 @@
     if ((strstr (config, "wvLaTeX.xml") != NULL)
 	|| (strstr (config, "wvCleanLaTeX.xml") != NULL))
 	printf
-	    ("\n\\resizebox*{\\baselineskip}{!}{\\includegraphics{placeholder.eps}}\
- 		  \n-- %#.2x graphic: StrangeNoGraphicData --",
+	    ("\n-- %#.2x graphic: StrangeNoGraphicData --",
 	     graphicstype);
     else
 	printf ("<img alt=\"%#.2x graphic\" src=\"%s\"%s><br%s>", graphicstype,
